packages/design-system/src/components/button/stories/index.js

```diff
@@ -228,36 +212,40 @@ export const ToggleButtons = () => {
       <ToggleButtonContainer
         isToggled={isToggled}
         swapToggled={swapToggled}
+        {...args}
         type={BUTTON_TYPES.QUATERNARY}
       />
       <DarkThemeProvider>
         <ToggleButtonContainer
           isToggled={isToggled}
           swapToggled={swapToggled}
+          {...args}
           type={BUTTON_TYPES.QUATERNARY}
         />
       </DarkThemeProvider>
     </>
   );
 };
+ToggleButtons.parameters = { controls: { include: [] } };
 
-export const PrebakedButtons = () => {
+export const PrebakedButtons = (args) => {
```

Comment:
Same here: do we need controls here? Maybe we could remove the `args` argument from this story and add it later if needed.
![image](https://user-images.githubusercontent.com/22185279/151239770-10884161-f732-40f8-81fb-6ddae01ed763.png)

Reply:
same reason. Args are passed in so we can filter them out and not show `controls` that do nothing.